client/src/pages/contact.tsx

```diff
@@ -12,18 +12,17 @@ interface ContactPageProps {
 
 const ContactPage = ({location}: ContactPageProps) => {
   const generalemail = 'screeningtool.feedback@usds.gov';
```

Comment:
Can these two be camelCased?

Reply:
Good catch, it should be. (again, seems like linter should be catching this?)

Reply:
fixed